### PR TITLE
Add legacy build variable

### PR DESCRIPTION
**Pull request by ann0see**
Related to https://github.com/jamulussoftware/jamulus/pull/1768

# Does this need translation?

- [ ] Yes 
- [x] No 

# Changes

Add a variable to allow direct download of legacy builds. While strictly this doesn't require translations; we might need to add other buttons somewhere (for the legacy build).

### CODE REVIEW

**Comment by ann0see** [comment]
Merged since the main PR by softins was merged too.

**Comment by jujudusud** [comment]
It is called 'mac-lagacy' ?

**Comment by ann0see** [comment]
No. `jamulus_version_mac_legacy.dmg`

**Comment by ann0see** [comment]
For me it's jamulus_3.8.0rc1dev-8789c32_mac_legacy.dmg

**Comment by jujudusud** [comment]
Capture d’écran du 2021-05-28 16-21-45

**Comment by ann0see** [comment]
Ah. You mean the variable. Yes, that's mac-legacy ;-)

**Comment by jujudusud** [comment]
Should be, but for now it is not ... and it is merged ... :-(

File: _config.yml [hunk 1]
```diff
@@ -15,6 +15,7 @@ download_file_names:
   deb-headless:              "jamulus_headless_3.7.0_ubuntu_amd64.deb"
   windows:                   "jamulus_3.7.0_win.exe"
   mac:                       "jamulus_3.7.0_mac.dmg"
+  mac-lagacy:                "jamulus_3_7_0_mac_legacy.dmg"
```
Comment (jujudusud) [inline _config.yml:18]: Should be:
`mac-legacy`

Comment (hoffie) [reply _config.yml:18]: > Should be:
> `mac-legacy`

Hrm, why?

https://github.com/jamulussoftware/jamulus/pull/1768/files#diff-55d631a22c9e350832273ee0abf8a6013435fa6051789cbe311b981b2ec0f348R132
https://github.com/jamulussoftware/jamulus/pull/1768/files#diff-ff6c0967f1173b1c4de6ea1230b43caaebe9c69607ca8e6dbb4b37f5ee3c6ec3R6

**Comment by hoffie** [comment]
> Ah. You mean the variable. Yes, that's mac-legacy ;-)

`-` in yaml keys looks a bit strange, but other targets (`deb-headless`) use it as well, so it should work.

**Comment by ignotus666** [comment]
I think he means the spelling: l**a**gacy vs l**e**gacy

**Comment by hoffie** [comment]
> I think he means the spelling: l**a**gacy vs l**e**gacy

Aaah! I agree. Thanks @jujudusud (for pointing it out) and @ignotus666 (for clarification). Yes, this should be adapted. Can someone provide a fix?

**Comment by jujudusud** [comment]
Will do

**Comment by ann0see** [comment]
Uh… now I understand ;-(. Happy if you fix it @jujudusud

**Pull request merged**